### DISCUSSION TITLE
引用外部库时如何使用预编译头文件的功能

### DISCUSSION THREAD

**Discussion by TOMO-CAT** [xmake-io/xmake#5364] [Q&A]
### 你在什么场景下需要该功能？

当前官方文档对于头文件预编译的内容比较简略：
![image](https://github.com/user-attachments/assets/e6c6ed29-2092-4467-8216-876614320ff3)
假设我需要引用多个 package 的多个头文件，我希望这些头文件都能预编译，现在应该怎么做呢？

### 描述可能的解决方案

暂无

[a comment by an automated account is not shown]

**Comment by waruqi** [comment]
自己搞个 header.h 去 include 所有你想要预编译的 headers 然后把这个 header.h 做成预编译 不就好了么。

[a comment by an automated account is not shown]

**Comment by TOMO-CAT** [comment]
> 自己搞个 header.h 去 include 所有你想要预编译的 headers 然后把这个 header.h 做成预编译 不就好了么。

好的，我看看 xmake test 里的例子。

[a comment by an automated account is not shown]
